Services/SwissBitcoinPayService.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Net;
+using System.Net.Http;
+using System.Security.Cryptography;
+using System.Text;
+using Newtonsoft.Json;
+using SmartStore.SwissBitcoinPay.Models;
+using SmartStore.SwissBitcoinPay.Settings;
+
+namespace SmartStore.SwissBitcoinPay.Services
+{
+    public class SwissBitcoinPayService
+    {
+
+        public static bool CheckSecretKey(string key, string message, string signature)
+        {
+            var msgBytes = HMACSHA256.HashData(Encoding.UTF8.GetBytes(key), Encoding.UTF8.GetBytes(message));
+            string hashString = string.Empty;
+            foreach (byte x in msgBytes)
+            {
+                hashString += String.Format("{0:x2}", x);
+            }
+            return (hashString == signature);
+        }
+
+        public string CreateInvoice(SwissBitcoinPaySettings settings, PaymentDataModel paymentData)
+        {
+
+            try
+            {
+                var invoice = new SwissBitcoinPayInvoiceModel()
+				{
+                    title = paymentData.Description,
+                    description = $"{paymentData.BuyerName} | Order : {paymentData.OrderID} | Store : {paymentData.StoreID.ToString()}",
+                    unit = paymentData.CurrencyCode,
+					amount = paymentData.Amount,
+                    email = paymentData.BuyerEmail,
+                    emailLanguage = paymentData.Lang,
+                    redirectAfterPaid = paymentData.RedirectionURL,
+                    webhook = paymentData.WebHookURL,
+                    delay = 60,
+                    onChain = settings.AcceptOnChain,
+                    extra = new SwissBitcoinPayExtraModel()
+                    {
+                        CustomNote = $"Order {paymentData.OrderID}"

```

Comment:
Doit être en camelCase :)

```suggestion
                        customNote = $"Order {paymentData.OrderID}"

```